test/unit/reporters/apiary-reporter-test.coffee

```diff
@@ -356,6 +356,56 @@ describe 'ApiaryReporter', () ->
             assert.isFalse call.isDone()
             done()
 
+    describe 'when adding skipped test', () ->
+      call = null
+      runId = '507f1f77bcf86cd799439011'
+      clonedTest = null
+      requestBody = null
+
+      beforeEach () ->
+        clonedTest = clone(test)
+        test.status = 'skip'
```

Comment:
I'd prefer to use `testOrigin` be untouched, and `test = clone(testOrigin)` :+1: